src/scss/components/_hero-section.scss

```diff
@@ -73,7 +73,18 @@
   &__cloud {
     filter: $svg-cloud-shadow;
   }
-}
+
+  &__cloud-animation-right {
+    animation: right 200s linear infinite;
+  	}
+
+    @keyframes right {
+    		to {
+        transform: translateX(2000px);
```

Comment:
should we use `px` here?

Reply:
Yeah, that feels magic numbery to me, even if this were `em` or `rem`. Why 2000? Also, formatting needs some attention.

Reply:
@zastrow That was the number we can up with when we paired last. It felt right for the animation so I kept it like that with an overflow hidden in the container. If there's a better way of doing this or a number that makes more sense let me know :)